src/test/test_step.go

```diff
@@ -384,8 +384,8 @@ func doTestResults(tid int, state *core.BuildState, target *core.BuildTarget, ru
 		}
 	} else {
 		var stdout []byte
-		stdout, err = prepareAndRunTest(tid, state, target, run)
```

Comment:
Oooff and now the value of the variable called `stderr` from above is stored in a variable called `stdout`... This is very confusing.